src/Engine/Graphics/Outdoor.cpp

```diff
@@ -1852,6 +1852,7 @@ void OutdoorLocation::subconstuctor() {
     // DLVHeader::DLVHeader(&v1->ddm);
 }
 
+// TODO(pskelton): Move to party?
```

Comment:
Party should be passed as a parameter really, otherwise i think it's good.
ODM_ProcessPartyActions is like a strategy pattern which is good for both unit testing and the separation of concerns